wordpress/wp-content/plugins/memberful-wp/src/metering/storage.php

```
@@ -13,61 +21,136 @@ class Memberful_Metering_Storage {
+
+    if ( null === $subject ) {
+      $subject = bin2hex( random_bytes( self::SUBJECT_BYTES ) );
+      $expires = time() + ( max( 1, $period_days ) * DAY_IN_SECONDS );
+      $signed  = self::sign( $subject, $expires );
+
+      self::send_cookie( $signed, $expires );
+      $_COOKIE[ self::COOKIE_NAME ] = $signed;
+    }
+
+    return self::apply_subject_filter( $subject );
+  }
```

PHPMD (2.15.0)

[error] get_or_create_subject accesses the super-global variable $_COOKIE. (undefined)

(Superglobals)